cumulus_library/studies/discovery/code_definitions.py

```diff
@@ -0,0 +1,53 @@
+# A collection of codes & codeableConcepts to extract available codes from.
```

Comment:
Philosophizing: I'm starting to get to the point over here where I'm considering more general seperation of table business logic from the bits that do the building. Somewhat releated, I created this: #132 - looking for a temperature read about this generally.

Reply:
I'm not 100% understanding your options here, maybe an in-person rubber-ducking would help. Lunch talk or something.